### PR TITLE
Using the new lighthouse in the framework

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**
- Removing influxdb and old way of installing lighthouse as it won't
  work with the new Portworx versions
- Not handling certs needed for https, removing the params from config.json

### CODE REVIEW

**Comment by piyush-nimbalkar** [comment]
The password in the config is only used in the readiness check. I think we can skip the readiness check. That way the users will not have to update the password in the framework if they change in the GUI.
Otherwise on restarts, they will have keep getting the benign`Invalid credentials` error from the readiness check.
@disrani-px what do you think?

File: frameworks/portworx/universe/config.json [hunk 1]
```diff
@@ -291,39 +244,46 @@
...
           "webui_port": {
             "title": "WebUI Port",
-            "description": "",
+            "description": "Port on which Lighthouse can be accessed",
             "type": "integer",
             "default": 8085
           },
+          "webui_ssl_port": {
+            "title": "WebUI HTTPS Port",
+            "description": "Port to access Lighthouse if you are running it on HTTPS",
```
Comment (disrani-px) [inline frameworks/portworx/universe/config.json]: "... if you are accessing it via HTTPS"

**Review by disrani-px — APPROVED** [review]

**Comment by disrani-px** [comment]
@piyush-nimbalkar Yes, let's not use a readiness check with the password then. The readiness check can just hit some lightouse endpoint and return ok if it doesn't get a 404 or timeout

File: frameworks/portworx/src/main/dist/svc.yml [hunk 2]
```diff
@@ -240,57 +204,62 @@ pods:
...
+              scheme: http
+              clientcertificates:
+                key: ''
+                cert: ''"> $MESOS_SANDBOX/lh-data/config.yaml
+            fi
+
+            LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu /entry-point.sh -confpath $MESOS_SANDBOX/lh-data -http_port {{LIGHTHOUSE_HTTP_PORT}} -https_port {{LIGHTHOUSE_HTTPS_PORT}}
         readiness-check:
-          cmd: LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu curl -X GET http://lighthouse-0-start.${FRAMEWORK_HOST}:{{LIGHTHOUSE_WEBUI_PORT}}/api/system | grep Lighthouse | grep 2
+          cmd: LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu curl -s -u {{LIGHTHOUSE_ADMIN_USER}}:{{LIGHTHOUSE_ADMIN_PASSWORD}} http://$TASK_NAME.$FRAMEWORK_HOST:{{LIGHTHOUSE_HTTP_PORT}}/api/manageCluster/list?companyId=1
```
Comment (disrani-px) [inline frameworks/portworx/src/main/dist/svc.yml]: Does lighthouse work with http even when https is enabled?

Comment (piyush-nimbalkar) [reply frameworks/portworx/src/main/dist/svc.yml]: It redirects to https. The readiness check may not work. But currently we do not provide a way to enable LH on HTTPS. Will fix the scheme when we add a way to provide certificates.

**Review by disrani-px — APPROVED** [review]

**Pull request merged**